### ISSUE TITLE
Allow alpha in custom fog color, or other way to adjust opacity

### ISSUE THREAD

**Issue by Miclebrick** [sp614x/optifine#1538]
This would allow resource packs to adjust how thick fog is or disable it in certain dimensions

**Comment by realjackasterisk** [comment]
One of my older ~~and admittedly, badly written~~ posts addresses this. Still waiting on it though. #1052

**Comment by Miclebrick** [comment]
I don't think having an issue with a "huge list of suggestions" is prudent. Maybe you should close it and split it into individual issues.

**Comment by realjackasterisk** [comment]
Yeah, you're right. I've been thinking about that for a while, I'll do it now.

**Comment by realjackasterisk** [comment]
There we go: #1542

**Comment by Miclebrick** [comment]
The reason I want this, is because with fancy fog, things that are still in render distance are simply unviewable: https://i.imgur.com/RcYT3pO.jpg
You can't see anything but white, maybe a general shape.

**Comment by sp614x** [comment]
@Miclebrick this is the idea of fog, it hides things so they are not visible anymore.

**Comment by Miclebrick** [comment]
Yeah :P But I like the effect on the air it has when you put the radius down, it makes it look almost like you have shaders on.

**Comment by Miclebrick** [comment]
https://i.imgur.com/3msfnzc.jpg If I didn't take the screenshot myself I almost wouldn't believe shaders were not used

**Comment by sp614x** [comment]
You can do this by changing "Video Settings -> Details -> Fog Start"

**Comment by Miclebrick** [comment]
If the distance is too high, it doesn't have the same effect. If the distance is too low, you can't see anything. I've tried the different settings and there isn't really a balance to be found, there's just too high and too low

**Comment by sp614x** [comment]
See here: https://github.com/sp614x/optifine/issues/1542#issuecomment-405183629

**Comment by Miclebrick** [comment]
![image](https://user-images.githubusercontent.com/6323125/42759362-b21f41dc-88d4-11e8-9c33-e3dda94dbfde.png)
0.8

https://i.imgur.com/tPgmDvg.jpg
0.6

https://i.imgur.com/hfYGlju.jpg
0.4


https://i.imgur.com/hfYGlju.jpg
0.2

0.2 looks amazing at short range
